mkosi.build

```diff
@@ -20,11 +20,15 @@
 # This is a build script for OS image generation using mkosi (https://github.com/systemd/mkosi).
 # Simply invoke "mkosi" in the project directory to build an OS image.
 
+# If mkosi.builddir/ exists mkosi will set $BUILDDIR to it, let's then use it
+# as out-of-tree build dir. Otherwise, let's make up our own builddir.
+[ x"$BUILDDIR" = x ] && BUILDDIR=build
```

Comment:
Oh, indeed. My bad. bash does a different thing in interactive mode and in script mode. I pasted some commands into a bash session, and did **does** terminate the session, but not when the same commands are part of a script. You're right, the code works fine. So maybe
```bash
[ -z "$BUILDDIR" ] && BUILDDIR=build
```
?